.github/workflows/winget.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         run: |
           version=$(echo "${{ github.event.release.tag_name }}" | grep -oP "(?<=v)[\d.]+")
           echo "version=$version" >> $GITHUB_OUTPUT
-      - uses: vedantmgoyal2009/winget-releaser@93fd8b606a1672ec3e5c6c3bb19426be68d1a8b0
+      - uses: vedantmgoyal9/winget-releaser@b87a066d9e624db1394edcd947f8c4e5a7e30cd7
```

Comment:
Why change to the latest repo commit? It looks like the v2 commit hash is still the same? https://github.com/vedantmgoyal9/winget-releaser/releases/tag/v2

Reply:
Sorry for not explaining; the [readme](https://github.com/vedantmgoyal9/winget-releaser/blob/main/README.md) now recommends using the `main` branch as it is using the latest version of [Komac](https://github.com/russellbanks/Komac) (ported to rust). `v2` hasn't been updated for a year now, and it's causing workflow failures (https://github.com/vedantmgoyal9/winget-releaser/issues/304).

Reply:
ah, got it - thanks!